.github/workflows/main.yml

```diff
@@ -39,5 +39,8 @@ jobs:
         skip-pkg-cache: true
 
     - name: Go Test
-      run: |
-        go test -v -vet=off ./...
+      run: go test -race -coverprofile=coverage.txt -covermode=atomic $(go list ./pkg/...)
```

Comment:
We can only test ./pkg right now, but we may need to test other pkg as well

Reply:
We can test other directories at that time.